src/main/java/rs/teslaris/core/harvester/product/Product.java

```diff
@@ -0,0 +1,56 @@
+package rs.teslaris.core.harvester.product;
+
+import java.util.List;
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlElementWrapper;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlType;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import rs.teslaris.core.harvester.common.MultilingualContent;
+import rs.teslaris.core.harvester.common.PersonAttributes;
+
+@XmlType(name = "TProduct", namespace = "https://www.openaire.eu/cerif-profile/1.1/")
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlRootElement(name = "Product")
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+public class Product {
+
+    @XmlElement(name = "Keyword")
+    List<String> keywords;
+
+    @XmlAttribute(name = "id")
+    private String id;
+
+    @XmlElement(name = "Type", namespace = "https://www.openaire.eu/cerif-profile/vocab/COAR_Product_Types")
+    private String type;
+
+    @XmlElement(name = "Language")
+    private String language;
+
+    @XmlElement(name = "Name")
+    private MultilingualContent name;
+
+    @XmlElement(name = "URL")
+    private MultilingualContent url;
+
+    @XmlElement(name = "Description")
+    private String description;
```

Comment:
Probably List<MultilingualContent>

Reply:
Description of MultilingualContent makes more sense to me :smile: